src/api/Rakefile

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env rake2.2
+#!/usr/bin/env rake2.3
```

Comment:
OBS already runs on 2.3. This probably was missed when upgrading ruby, and rake.